### PR TITLE
Add entry for boomerang

### PR DESCRIPTION
**Pull request by bluesmoon**
This adds boomerang detection and version extraction.
Requesting review from @nicjansma

### CODE REVIEW

**Comment by nicjansma** [comment]
Do we add an entry in the README too?

File: library/libraries.js [hunk 1]
```diff
@@ -1171,5 +1171,15 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
             }
             return false;
         }
+    },
+    'boomerang.js': {
+        icon: 'icon_48', // currently has no icon
+        url: 'https://soasta.github.io/boomerang/doc/',
+        test: function(win) {
+            if (win.BOOMR) {
```
Comment (johnmichel) [inline library/libraries.js]: In addition to the check for `window.BOOMR`, please also use one of the non-version properties from the `BOOMR` object when checking for existence of boomerang.

**Comment by bluesmoon** [comment]
@nicjansma @johnmichel I've addressed both your requests.

**Comment by johnmichel** [comment]
LGTM

Thanks!

**Pull request merged**